src/config/oracle.call.ts

```diff
@@ -0,0 +1,54 @@
+import { ThirdwebSDK } from "@thirdweb-dev/sdk";
+import { OracleData } from "@/abis/oracle.data.check";
+
+const eventName = "SentMessage";
+
+const sdk = ThirdwebSDK.fromPrivateKey(
+  process.env.NEXT_PUBLIC_PRIVATE_PAPANTLA_KEY as string, // Your wallet's private key (only required for write operations)
+  "Goerli" // Optimism
+);
+
+export const oracleConnection = async () => {
+  // Connect to your smart contract using the contract address
+  const contract = await sdk.getContract(
+    "0x5837d7635e7E9bf06245A75Ccd00A9a486Dd0b72",
+    OracleData as any
+  );
+  return contract;
+};
+
+// getAssertionResult Mumbai contract 0x0000000000000000000000005837d7635e7e9bf06245a75ccd00a9a486dd0b72
+// 1
+export const execAssertTruth = async (flightId: string) => {
+  const contract = await oracleConnection();
+  const { receipt } = await contract.call("assertTruth", "YA60B");
```

Comment:
Second parameter of should be flightId, not "YA60B":

const { receipt } = await contract.call("assertTruth", flightId);